robot_smach_states/src/robot_smach_states/navigation/navigation.py

```diff
@@ -219,7 +229,7 @@ def __init__(self, robot, reset_head=True, speak=True, reset_pose=True, input_ke
 
             with sm_nav:
 
-                smach.StateMachine.add('GET_PLAN',                          getPlan(self.robot, self.generateConstraint, self.speak),
+                smach.StateMachine.add('GET_PLAN',                          getPlan(self.robot, constraint_function, self.speak),
                     transitions={'unreachable'                          :   'unreachable',
                                  'goal_not_defined'                     :   'goal_not_defined',
                                  'goal_ok'                              :   'EXECUTE_PLAN'})
```

Comment:
Not sure why people add such large amounts of white space, but it existed already

Reply:
Now is as good a time as any to clean it up.